js/src/duplicate-post-edit-script.js

```diff
@@ -11,11 +11,26 @@ import { __ } from "@wordpress/i18n";
  */
 const render = () => (
 	<PluginPostStatusInfo>
-		{ duplicatePostRewriteRepost.permalink !== "" &&
-		  <a href={ duplicatePostRewriteRepost.permalink }>{ __( "Rewrite & Republish", "duplicate-post" ) }</a> }
+		{ duplicatePostRewriteRepost.permalink !== '' &&
+		  <a href={ duplicatePostRewriteRepost.permalink }>{ __( 'Rewrite & Republish', 'duplicate-post' ) }</a> }
 	</PluginPostStatusInfo>
 );
 
-registerPlugin( "duplicate-post", {
+registerPlugin( 'duplicate-post', {
 	render
 } );
+
+if ( parseInt( duplicatePostRewriteRepost.rewriting) ) {
+	(function (wp) {
```

Comment:
There shouldn't be a space between "function" and "(wp)", please change to `function(wp)`. 
Note: unclear what coding standards should be used for JS, e.g. spaces within parenthesis. Will ignore for now.